src/main/java/core/basesyntax/figure/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,52 @@
+package core.basesyntax.figure;
+
+public class IsoscelesTrapezoid extends Figure {
+    private double base1;
+    private double base2;
+    private double height;
+
+    public IsoscelesTrapezoid(String color, double base1, double base2, double height) {
+        super(color);
+        this.base1 = base1;
+        this.base2 = base2;
+        this.height = height;
+    }
+
+    public double getBase1() {
+        return base1;
+    }
+
+    public void setBase1(double base1) {
+        this.base1 = base1;
+    }
+
+    public double getBase2() {
+        return base2;
+    }
+
+    public void setBase2(double base2) {
+        this.base2 = base2;
+    }
+
+    public double getHeight() {
+        return height;
+    }
+
+    public void setHeight(double height) {
+        this.height = height;
+    }
+
+    @Override
+    public double getArea() {
+        return height * ((base1 + base2) * 0.5);
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: " + getClass().getSimpleName() + ", area: "
```

Comment:
```suggestion
        System.out.println("Figure: IsoscelesTrapezoid, area: "
```
You can just put the name here, don't make it so complicated because the name of the class won't change. Fix it everywhere